### DISCUSSION TITLE
Buffering issue

### DISCUSSION THREAD

**Discussion by markjoren7** [kitadai31/revanced-patches-android6-7#37] [General]
i followed your process and created revenced ext apk for my android box 7.1 and it was working smoothly before but now i am getting buffering issue in my android media box 7.1 plz advice.

**Comment by kitadai31** [comment]
fixed in [revanced-patches-2.160.5](https://github.com/kitadai31/revanced-patches-android6-7/releases/tag/v2.160.5)
